lib/phoenix/socket/transport.ex

```diff
@@ -201,10 +202,11 @@ defmodule Phoenix.Socket.Transport do
     if channel = socket.handler.__channel__(topic, socket.transport_name) do
       socket = %Socket{socket | topic: topic, channel: channel}
 
+      filtered_payload = Instrument.filter_values(msg.payload, Application.get_env(:phoenix, :filter_parameters))
```

Comment:
Would it be a bad pattern having the attribute declared in the `Instrument` module and then calling `Instrument.filter_values(msg.payload)`?